### PR TITLE
Block Controls

### PR DESCRIPTION
**Pull request by Duzos**
For #269

### CODE REVIEW

File: src/main/java/loqor/ait/core/entities/TardisRealEntity.java
Comment (Duzos) [inline src/main/java/loqor/ait/core/entities/TardisRealEntity.java:1]: You didnt want this feature..
Right?

Comment (Loqor) [reply src/main/java/loqor/ait/core/entities/TardisRealEntity.java:1]: well i do now

Comment (Duzos) [reply src/main/java/loqor/ait/core/entities/TardisRealEntity.java:1]: okay ill fix it

**Comment by Duzos** [comment]
Review not required - Loqor says LGTM privately

**Comment by Duzos** [comment]
After this PR @Loqor 
- A proper implementation of the block control

**Pull request merged**